Tasks/VSMobileCenterTest/task.json

```diff
@@ -210,6 +222,7 @@
         },
         {
             "name":"xcuitestBuildDir",
+            "aliases": [ "xcUITestBuildDirectory" ],
```

Comment:
prefix doesn't match the next one. What does xc stand for? Seems like an abbreviation.